cfgov/unprocessed/css/enhancements/tables.less

```diff
@@ -120,44 +120,36 @@
         }
     }
 
-    .respond-to-max(960px, {
-
-        font-size: 14px;
-
-        thead {
-            display: none;
-        }
+}
 
-        tbody {
-            border-top: none;
+.respond-to-max( @bp-xs-max, {
+    .o-table__stack-on-small {
+        tr,
+        td {
+            display: block;
         }
 
-        tr {
-            border-bottom: none;
+        & > thead {
+          display: none
         }
 
-        td:first-child {
-            .h4();
-            margin-bottom: 10px;
-            background-color: @td-bg-mobile;
+        td[data-label]:before {
+            .webfont-demi();
+            display: block;
+            margin-top: 0;
+            margin-bottom: .41666667em;
+            content: attr(data-label);
+            font-size: .875em;
```

Comment:
Update this to `unit( @size-v / @base-font-size-px, em );` so it's clear where it's coming from and will update automatically if that value ever changes.